src/modules/menu/components/search/SearchBar.vue

```diff
@@ -68,7 +68,6 @@ const clearSearchQuery = () => {
 
 const closeSearchResults = () => {
     showResults.value = false
-    searchInput.value.focus()
```

Comment:
@LukasJoss this should not have been removed, for user that use the keyboard if they go to a loaction the focus should stay on the search input so that they cans still change the search result

Reply:
Currently focusing the search bar while containing text immediately opens the dropdown again, hiding the position. What approach would you suggest to fix that (or should it be a post prod ticket since the current implementation is as in the old viewer?)